python/semantic_kernel/memory/memory_record.py

```diff
@@ -119,3 +115,27 @@ def local_record(
             timestamp=timestamp,
             embedding=embedding,
         )
+
```

Comment:
Are these schema changes that are required just for Mongo vCore?

If yes, will likely want to isolate these so they only apply for that case. @dmytrostruk what do you think?

Reply:
These are not schema changes, I just created getters for the variables that were not already there. It is better to access the value of the variables using getters instead of directly calling the variables. 

Across the SK code in other memoryStore the values of the variables are getting accessed directly, which is bad coding practice. SK team can update the code accordingly once this PR goes in.